frontend/src/employee-frontend/components/child-information/placements/CreatePlacementModal.tsx

```diff
@@ -128,13 +127,7 @@ function CreatePlacementModal({ childId, reload }: Props) {
       resolve={{
         action: submitForm,
         label: i18n.common.confirm,
-        disabled: errors.length > 0 || submitting || form.ghostUnit,
-        info: form.ghostUnit
-          ? {
-              text: i18n.childInformation.placements.warning.ghostUnit,
-              status: 'warning'
-            }
-          : undefined
+        disabled: errors.length > 0 || submitting || form.ghostUnit
```

Comment:
`|| form.ghostUnit` is redundant since it also adds to validation errors?

Reply:
Ah, yes, forgot to remove it.